winton_kafka_streams/processor/serialization/string.py

```diff
@@ -9,7 +9,7 @@ def __init__(self):
         self.on_error = 'strict'
 
     def serialize(self, topic, data):
-        return data.encode(self.encoding, self.on_error)
+        return str(data).encode(self.encoding, self.on_error)
```

Comment:
Hmm, this is to force int to become a str? Does this work properly on py3 with new unicode?

Reply:
Yes, I've only tested it on py3. It is a noop on an existing utf8 str.